mobile/src/main/java/org/openhab/habdroid/util/Constants.java

```diff
@@ -26,5 +26,19 @@
     public static final String PREFERENCE_TONE              = "default_openhab_alertringtone";
     public static final String PREFERENCE_SSLCLIENTCERT     = "default_openhab_sslclientcert";
     public static final String PREFERENCE_SSLCLIENTCERT_HOWTO = "default_openhab_sslclientcert_howto";
+    public static final String PREFERENCE_DEBUG_MESSAGES      = "default_openhab_debug_messages";
     public static final String DEFAULT_GCM_SENDER_ID        = "737820980945";
-}
+
+    public interface MESSAGES {
```

Comment:
I'm not sure if these values could be saved in a separate file? That would remove the requirement of using COnstants-MESSAGES.LOGLEVEL.NO_DEBUG, which, at least for me, looks pretty long :)

Reply:
If you just type "ALWAYS" Android Studio asks to make a static import.